doc/docs/modules/ROOT/pages/quickstart_writer.adoc

```diff
@@ -319,4 +319,107 @@ You can specify 3 different modes to use for saving the nodes:
 
 * `Overwrite`: will perform a `MERGE` on that node
 * `ErrorIfExists`: will perform a `CREATE`
-* `Match`: will perform a `MATCH`
\ No newline at end of file
+* `Match`: will perform a `MATCH`
+
```

Comment:
Not the right place here, but somewhere in the docs you need to document the "script" option that you added to this PR since it was opened.

Reply:
You can find the documentation here:
https://github.com/neo4j-contrib/neo4j-spark-connector/blob/3b687d6dd0f130d90b12e7fcf29ef5d200d5ffd1/doc/docs/modules/ROOT/pages/quickstart_writer.adoc#script-option